File: one/one.py
import sys


def one(limit):
    sum = 0
    for i in range(limit):
        if i % 3 == 0 or i % 5 == 0:
            sum += i
    return sum


if __name__ == '__main__':
    if len(sys.argv) != 2:
        print("Usage python3 one.py <limit>")
        sys.exit(1)

    limit = int(sys.argv[1])
    print(one(limit))

File: one/one.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv) {
    int limit;
    long sum;
    int i;

    if(argc != 2) {
        cout << "Usage: one <limit>" << endl;
        return 1;
    }

    limit = stoi(argv[1]);

    sum = 0;
    for(i=1; i<limit; i++) {
        if(i % 3 == 0 || i % 5 == 0) {
            sum += i;
        }
    }

    cout << sum << endl;

    return 0;
}
